src/components/MaterialFAQ.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <details>
+        <summary class="summary">
+            Ich kann die Aufgabe schon alleine lösen, bin ich jetzt vergebens
+            hier?
+        </summary>
+        Nein! Ihr könnt gerne aufpassen, dass ich keinen Fehler mache und ggf.
+        euren eigenen Code verbessern.
+    </details>
+
+    <details v-if="hasCommand">
+        <summary class="summary">Was macht der Befehl überhaupt?</summary>
+        Dieser Befehl lädt Code herunter und führt diesen direkt aus, sodass für
```

Comment:
This sentence isn't intuitive to read. I'd suggest the following: **Dieser Befehl lädt Code herunter und führt diesen direkt aus. In diesem Fall wird ein ZIP-Archiv heruntergeladen und gleich entpackt. Das ZIP-Archiv wird im Nachhinein wieder gelöscht, sodass nur noch der entdeckte Inhalt übrig bleibt.**